javascript/package-lock.json

```diff
@@ -0,0 +1,1736 @@
+{
```

Comment:
The npm lockfile is verbose, but in JavaScript we usually version it to ensure everyone can install the same packages easily.